src/examples/usage.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-var DarkcoinClient = require("darkcoin-client").default;
+var DarkcoinClient = require("../../build/darkcoin-client").default;
```

Comment:
You likely don't need this change even in development. `npm install or link` should be enough.

Reply:
oh, nevermind. This is good.

Reply:
Eh to be honest, I don't know if it ever really worked like this. But if it works for you then cool, maybe we do leave it that way.